video/out/wayland_common.c

```diff
@@ -225,7 +226,7 @@ static void surface_handle_enter(void *data,
         }
     }
 
-    wl->window.events |= VO_EVENT_WIN_STATE;
+    wl->window.events |= VO_EVENTS_USER;
```

Comment:
`VO_EVENTS_USER` is just a mask of events that should be propagated to the playback core (instead of just vo.c or e.g. vo_opengl.c), and you shouldn't use it directly. `VO_EVENT_WIN_STATE` is used for updating a group of windows-related properties that do not update often (i.e. not during normal playback while the user is not interacting with the window). `VO_EVENT_RESIZE` is supposed to happen when the reported window size changes (I'm not sure why it isn't just part of `VO_EVENT_WIN_STATE`.)

`VO_EVENTS_USER`  _could_ be used as catch-all change notification, but normally more fine-grained events would be preferable. It's also possible that something more expensive could be added to the mask.